pkg/util/runtime/cpu_linux.go

```diff
@@ -38,13 +38,21 @@ import (
 func NumCPU() int {
 	cpus := runtime.NumCPU()
 
-	cgroupPath, err := libcontainercgroups.FindCgroupMountpoint("", "cpu")
 	if err != nil {
```

Comment:
Is this required ???

Reply:
The [cgroups documentation](https://man7.org/linux/man-pages/man7/cgroups.7.html) says that in cgroups v1, you can have controllers mounted in multiple points in the filesystem. This function gets the appropriate mountpoint for the current process, so it's necessary to keep it for proper v1 support (It was moved to the part of the code that specifically deals with cgroups v1). 

The function doesn't work for cgroups v2 as it implements a 'unified hierarchy' as seen in the error message in your initial message, which is why it was moved to the part of the code that handles parsing cgroups v1 files.  

Is there a reason why it may not be required?